components/brave_vpn/browser/connection/brave_vpn_os_connection_api.cc

```diff
@@ -264,4 +266,24 @@ void BraveVPNOSConnectionAPI::ToggleConnection() {
   can_disconnect ? Disconnect() : Connect();
 }
 
+void BraveVPNOSConnectionAPI::SetInstallSystemServiceCallback(
+    base::RepeatingClosure callback) {
+  install_system_service_callback_ = std::move(callback);
+}
+
+void BraveVPNOSConnectionAPI::InstallSystemServices() {
+  // This API could be called more than once because BraveVpnService is a
+  // per-profile service. If service install is in-progress now, just return.
```

Comment:
NOTE: As written, this will get called a lot.
This should be capped - for example, once per session (ex: each relaunch would reset).

Reply:
Fixed in https://github.com/brave/brave-core/pull/20754/commits/ce1c350181d15eb536f168a3cd5fb5f06962eb99